scripts/static_content_generation/Generate_StaticContent_MD.pl

```diff
@@ -0,0 +1,389 @@
+
+my @input_genome_reports = <${genome_report_dir}/*.genomereport.json>;
+
+## For each genome report file, locate and combine its assoicated Wilipedia JSON summary file on GCF/GCA accession
+system ("mkdir -p ./StaticContent_MD_Output-${release}");
+
+print BLUE "\nParsing JSON/Refseq files\n\n";
+
+foreach my $genome_sum_file (@input_genome_reports){
+
+	print DARK GREEN "Beginning processing RefSeq annotation:\n[$genome_sum_file]\n";
+
+	#Species specific static content folder
+	my $static_output_folder = "StaticContent_MD_Output-${release}";	
+	
+	## Control flow and other control variables
+	my $has_wiki_json_data=0;
+	my $wrap_at = 70; #Length of each paragraph line printed.
+	my $wiki_about;
+
+	#per core global vars
+	my ($scientific_name, $extract, $jpg_source, $wiki_url, $about_content, $refseq_annotation_content, $community_anno_content, $dtol_community_anno_content, $assembly_content);
+	
+	# Genome annotation report variables
+	my ($species_name, $common_name, $taxa_id, $anno_report_url, $asseb_accession, $release_version, @ncbi_rel, $annotation_source, $assembly_submitter);
+
+	# Convert GCF to GCA and locate its wiki JSON file
+	my ($refseq_file, $gca_accession, $core_prodname, $gca_accession_escape);
+
+	## Parse species name from ref_seq annotion
+	$species_name = `jq '.organism.organism_name' $genome_sum_file | sed 's/"//g'`;
+	chomp $species_name;
+	$scientific_name = $species_name;
+	$species_name =~ tr/ /_/;
+	$static_output_folder = $static_output_folder."/".$species_name;
+	my @temp_split_sp_name = split ("_", $species_name);
+
+	#Check whether or not the species name is bionomial or trinomial
+	if ((scalar @temp_split_sp_name == 3 ) && ($temp_split_sp_name[1] eq $temp_split_sp_name[2])){
+		print YELLOW "!!! Found trinomial species name instance !!!! \"$species_name\" --> Utilizing just the bionomial: ";
+		$species_name = "$temp_split_sp_name[0]_$temp_split_sp_name[1]";
+		print "\"$species_name\"\n";
+	}
+	else{
+		print "Species name: \"$scientific_name\".\n";
+	}
+
+	$species_name = lc$species_name;
+	system ("mkdir -p ./$static_output_folder");
+
+	# Obtained common name and taxon_id information from genome report file
+	$common_name = `jq '.organism.common_name' $genome_sum_file | sed 's/"//g'`;
+	chomp $common_name;
+	$taxa_id = `jq '.organism.tax_id' $genome_sum_file`;
+	chomp $taxa_id;
+
+	# Try to obtain NCBI refseq release URL, this will only be populated if there is an associated RefSeq annotation
+	# If not, likely the genome is community annotated instead i.e. GFF3 submitted with assembly to Genbank.
+	$anno_report_url = `jq '.annotation_info.report_url' $genome_sum_file | sed 's/"//g'`;
+	chomp $anno_report_url;
+
+	# Import check whether or not the assembly is linked to DToL:
+	my $dtol_linked = 0;
+	my $sanger_assembly = 0;
+	my $dtol_project_acc = "PRJEB40665"; # https://www.ncbi.nlm.nih.gov/bioproject/PRJEB40665/
```

Comment:
@leannehaggerty would this fall apart ? Im using what I believe is the main DToL project Accession to identify DToL genome assemblies.

Reply:
This should be fine, we use the same accession. If an assembly that should be part of the DToL project does not fall under this umbrella accession then I would say that is on the submitter :)

Reply:
Ahh great !